library/testing/TestSDKDeprecatedOptions.cxx

```diff
@@ -0,0 +1,45 @@
+#include <engine.h>
+#include <options.h>
+#include <scene.h>
+#include <window.h>
+
+#include "TestSDKHelpers.h"
+
+#if defined(_MSC_VER)
+#pragma warning(push)
+#pragma warning(disable : 4996)
+#elif defined(__clang__)
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated-declarations"
+#elif defined(__GNUC__)
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
+#endif
```

Comment:
use macros.h ?

Reply:
nop, `macros.h` is private and not accessible from tests.